security/cloudtrail.json

```diff
@@ -0,0 +1,134 @@
+{
+	"AWSTemplateFormatVersion": "2010-09-09",
+	"Description": "Security: CloudTrail across all regions, a cloudonaut.io template",
+	"Parameters": {
+		"CloudWatchLogsRetentionInDays": {
+			"Description": "The number of days log events are kept in CloudWatch Logs",
+			"Type": "Number",
+			"AllowedValues": [1, 3, 5, 7, 14, 30, 60, 90, 120, 150, 180, 365, 400, 545, 731, 1827, 3653],
```

Comment:
you can only select one of these values. http://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutRetentionPolicy.html#CWL-PutRetentionPolicy-request-RetentionInDays